include/ostatus.php

```diff
@@ -353,17 +353,25 @@ public static function import($xml,$importer,&$contact, &$hub) {
 				continue;
 			}
 
-			$item["body"] = add_page_info_to_body(html2bbcode($xpath->query('atom:content/text()', $entry)->item(0)->nodeValue));
 			$item["object-type"] = $xpath->query('activity:object-type/text()', $entry)->item(0)->nodeValue;
+			$item["verb"] = $xpath->query('activity:verb/text()', $entry)->item(0)->nodeValue;
+
+			if ($item["verb"] == ACTIVITY_POST && $xpath->evaluate('boolean(atom:summary)', $entry)) {
```

Comment:
According to my English teachers I guess I shouldn't even be able to order a beer in a bar. So sometimes I have to proof this :-)